velox/functions/prestosql/window/LeadLag.cpp

```diff
@@ -278,15 +284,26 @@ class LeadLagFunction : public exec::WindowFunction {
   std::vector<vector_size_t> rowNumbers_;
 };
 
-template <>
-void LeadLagFunction<true>::setRowNumbersForConstantOffset() {
+template <bool isLag>
+void LeadLagFunction<isLag>::setRowNumbersForConstantOffset() {
```

Comment:
Move the implementation to where the method is defined.

Reply:
done